README.md

```diff
@@ -1 +1,15 @@
-# hw01
\ No newline at end of file
+# hw01
+
+## Biography
+
+![Profile Image](/profile.jpg)
```

Comment:
Organizationally, this looks very clean. Well done leading with a photo, and good use of two headers at beginning